pytest_plugins/rerun_rp/rerun_rp.py

```diff
@@ -56,48 +85,63 @@ def pytest_addoption(parser):
         Defect_types:
             Collect failed tests marked with defect types: {[*ReportPortal.defect_types.keys()]}
         '''
-    parser.addoption("--only-failed", const='all', nargs='?', help=help_script)
+    parser.addoption("--only-failed", const='all', nargs='?', help=help_text)
+    help_text = '''
+        Collects only skipped tests in Report Portal to run only skipped tests.
```

Comment:
This needs updated for failed, not skipped. Needs to be more descriptive too - why would --only-failed be passed multiple times?

Reply:
Ah, I think I see what's going on, I was reading the diff wrong.

I think it would be more clear if the formatting was changed here, and the help text strings just passed directly to their respective `addoption` calls.

That, or use specific names for `help_text`, as visually this was confusing to parse.

Reply:
@mshriver Yeah earlier it was specific to the options but @mirekdlugosz has suggestion to override it!

Reply:
I'll hold my thoughts then :)